tests/api/test_backups.py

```diff
@@ -1035,6 +1035,9 @@ async def test_protected_backup(
     assert body["data"]["backups"][0]["location"] is None
     assert body["data"]["backups"][0]["locations"] == [None]
     assert body["data"]["backups"][0]["protected"] is True
+
+    # NOTE: Maybe it is not safe to compare size here, as random data in the
+    # backup might change the size (due to gzip).
```

Comment:
You could mention the slug/backup id, which is derived from the date, as well as the date itself will differ

Reply:
Improved the comment.